docs/sources/datasources/_index.md

```diff
@@ -91,6 +107,24 @@ These built-in core data sources are also included in the Grafana documentation:
 - [Testdata]({{< relref "./testdata" >}})
 - [Zipkin]({{< relref "./zipkin" >}})
 
+## Add additional data source plugins
+
+You can add additional data sources as plugins (that are not available in core Grafana), which you can install or create yourself.
+
+### Find data source plugins in the plugin catalog
+
+To view available data source plugins, go to the [plugin catalog](/grafana/plugins/?type=datasource) and select the "Data sources" filter.
+For details about the plugin catalog, refer to [Plugin management]({{< relref "../administration/plugin-management/" >}}).
```

Comment:
This link will need to be updated into a `docs/reference` style link like the others in the page because the linked "Administration" content doesn't exist in Grafana Cloud documentation.

For an example, refer to https://github.com/grafana/grafana/pull/83712/files#diff-7b86f2d305e1ba40bef37131931d4213ee72232f66190c6a31cc1de8b4d913ffR129-R130.